service/platform/index.js

```diff
@@ -243,6 +243,38 @@ class SoundEffectsApi {
     }
 }
 
+let webrtcvad;
+try {
+    webrtcvad = require('webrtcvad').default;
+} catch(e) {
+    console.log("VAD not available");
+    webrtcvad = null;
+}
+
+class VAD {
+    constructor() {
+        this._instance = null;
+    }
+
+    setup(bitrate, level) {
+        if (this._instance)
+            this._instance = null;
+
+        if (webrtcvad) {
+            this._instance = new webrtcvad(bitrate, level);
+            // console.log("setup VAD bitrate", bitrate, "level", level);
+            return true;
+        }
+
+        return false;
+    }
+
+    process(chunk) {
+        if (!this._instance)
+            return false;
+        return this._instance.process(chunk);
```

Comment:
Is this running synchronously in the nodejs thread? Should we use a Worker thread instead?